pkg/recovery/repair_test.go

```diff
@@ -229,19 +269,31 @@ func TestNewRepairHandler(t *testing.T) {
 			return nil, receiptError
 		})
 
-		// create the chunk repair handler
-		repairHandler := recovery.NewRepairHandler(mockStorer, logger, pushSyncService)
-
 		//create a trojan message to trigger the repair of the chunk
 		testTopic := trojan.NewTopic("foo")
-		maxPayload := make([]byte, swarm.SectionSize)
-		var msg trojan.Message
-		copy(maxPayload, c3.Address().Bytes())
-		msg, err = trojan.NewMessage(testTopic, maxPayload)
+		privKey, err := crypto.GenerateSecp256k1Key()
```

Comment:
but  why have  this test, to check if pushsync error is p
you do not  need this to test `recieve error`

Reply:
This test was written during normal chunk repair and i just modified it to adjust to instant chunk repair. I wanted to cover all error conditions during the test. i will remove it if you feel it is not required.